eShopsolution.Data/EF/EShopDbContext.cs

```diff
@@ -39,12 +40,24 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
                 entity.Property(e => e.ID).HasColumnType("int");
 
+            });
+            modelBuilder.Entity<Category>(entity =>
+            {
+                entity.ToTable("Category");
+
+                entity.Property(e => e.Id).HasColumnType("int");
```

Comment:
Do we need to add "entity.Property(e => e.Id).HasColumnType("int");" try to remove it and see if it's still working

Reply:
Có delete thử chưa, coi nó có tự động bind luôn hong